javascript/apps/taiga/src/app/shared/comments/comments.component.html

```diff
@@ -0,0 +1,115 @@
+<!--
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+Copyright (c) 2023-present Kaleidos INC
+-->
+
+<ng-container *transloco="let t">
+  <div class="header">
+    <h2 class="title">
+      {{ t('comments.title') }}
+      <span
+        *ngIf="total"
+        class="total-comments"
+        >({{ total }})</span
+      >
+    </h2>
+
+    <div
+      class="sort"
+      *ngIf="total">
+      <p
+        *ngIf="order === '-createdAt'"
+        aria-hidden="true">
+        {{ t('comments.order.new') }}
+      </p>
+
+      <p
+        *ngIf="order === 'createdAt'"
+        aria-hidden="true">
+        {{ t('comments.order.old') }}
+      </p>
+
+      <button
+        (click)="toggleOrder()"
+        type="button"
+        class="sort"
+        appearance="sort"
+        [attr.aria-label]="
```

Comment:
When I click the button to change comments order the whole view is refresh so you lose the scroll position. Is this behaviour correct?
![Peek 2023-06-12 08-25](https://github.com/taigaio/taiga/assets/5970959/612fd890-6dc8-487d-9b45-95b77d208822)